index.html

```diff
@@ -7,50 +7,48 @@
     <title>Commonplace Cooperative</title>
     <link rel="stylesheet" href="style.css">
   </head>
-  
-  <header>
-    <!-- Header section -->
-    <div class="header">
-      <!-- Nav Bar -->
-        <div class="nav-bar">
-           <div class="left-side"> <!--rename elements to <nav> ?-->
-            <img id="logo" src="/images/logo.png" alt="commonplace-logo">
-            <p id="commonplace-coop">commonplace.coop</p>
+  <body>
+    <header>
+      <!-- Header section -->
+      <div class="header">
+        <!-- Nav Bar -->
+          <div class="nav-bar">
+             <div class="left-side"> <!--rename elements to <nav> ?-->
+              <img id="logo" src="/images/logo.png" alt="commonplace-logo">
+              <p id="commonplace-coop">commonplace.coop</p>
+            </div>
+            <div class="right-side">
+              <nav class="site-routes">
+                <a href="#" id="services">services</a>
+                <a href="#" id="media">media</a>
+                <a href="#" id="events">events</a>
+                <a href="#" id="about">about</a>
+                <a href="#" id="join">join</a>
+              </nav>
+            </div>
           </div>
-          <div class="right-side">
-            <nav class="site-routes">
-              <a href="#" id="services">services</a>
-              <a href="#" id="media">media</a>
-              <a href="#" id="events">events</a>
-              <a href="#" id="about">about</a>
-              <a href="#" id="join">join</a>
-            </nav>
+        </div>
+      <!-- section 1 -->
+        <div class="white-space-one">
+          <div class="tagline-one">
+            <h3 id="commonplace">COMMONPLACE</h3>
+              <div class="text-right">
+                <p id="where">where</p> <!-- -->
```

Comment:
There's a lot of nesting in this markup. Sometimes it makes sense if we're working with reusable components, but we don't need to do that here. It makes it more difficult to understand (and style).